### PR TITLE
feat: optimize workflows

### PR DESCRIPTION
**Pull request by cobraz**
In order to keep GHA workflows updated, we need something automated. Fortunately, @dependabot can help us.

However, in our current setup. Dependabot cannot updated what we are storing in `templates/github/workflows`. This pull request changes the behaviour, so we've got the same workflows in `.github/workflows` as our templates. Dependabot updated them, and our weekly scheduled jobs copy them over to templates whenever there are changes. This way, our workflows are kept up to date.

### CODE REVIEW

**Comment by cobraz** [comment]
Not sure if we'd need this, let's wait until [this change](https://github.com/bjerkio/create-bjerk-typescript/commit/18f79e1c05e0d4aa4553e010ceb12b918cadc7b7) has had time to run for a while. Hopefully, we'll get a Dependabot PR 🤞

File: .github/workflows/weekly-cron-jobs.yaml [hunk 1]
```diff
@@ -50,3 +50,31 @@ jobs:
...
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: 'chore(deps): upgrade yarn to latest version'
+          author: Bjerk Bot <bjerk-bot@users.noreply.github.com>
+          token: ${{ secrets.BJERKBOT_GITHUB_TOKEN }}
+          branch: update-yarn
+          team-reviewers: thebranches
+          delete-branch: true
+          title: 'chore(deps): upgrade yarn to latest version'
```
Comment (cobraz) [inline .github/workflows/weekly-cron-jobs.yaml:73]:
```suggestion
          commit-message: 'chore(deps): update github actions workflows'
          author: Bjerk Bot <bjerk-bot@users.noreply.github.com>
          token: ${{ secrets.BJERKBOT_GITHUB_TOKEN }}
          branch: update-yarn
          team-reviewers: thebranches
          delete-branch: true
          title: 'chore(deps): update github actions workflows'
```

File: .github/workflows/weekly-cron-jobs.yaml [hunk 2]
```diff
@@ -50,3 +50,31 @@ jobs:
...
+      - run: cp .github/workflows/pull-request.yml templates/github/workflows/
+      - run: cp .github/workflows/workflow.yml templates/github/workflows/
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: 'chore(deps): upgrade yarn to latest version'
+          author: Bjerk Bot <bjerk-bot@users.noreply.github.com>
+          token: ${{ secrets.BJERKBOT_GITHUB_TOKEN }}
+          branch: update-yarn
```
Comment (cobraz) [inline .github/workflows/weekly-cron-jobs.yaml:70]:
```suggestion
          branch: update-gha-workflows
```

**Comment by cobraz** [comment]
I have intentionally left a few workflows out-of-date to see if there would be any Dependabot updates. I'm going to downgrade some of our workflows located in `.github` now to ensure that we have updates at all.

**Comment by cobraz** [comment]
I can verify that the actions are outdated at least:
- https://github.com/cobraz/actions-parsers/pull/1
- https://github.com/cobraz/actions-parsers/pull/2

**Review by braaar — APPROVED** [review]

**Review by braaar — APPROVED** [review]

**Pull request merged**